doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py

```diff
@@ -342,13 +342,16 @@ def wait_for_state_update(
         ##
         ## Ensuring Collision Updates Are Received
         ## ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-        ## If the Python node dies before publishing a collision object update message, the message
+        ## If the Python nodewas just created and did not connect to other nodes yet, or 
```

Comment:
Did you observe any specific issue with a _just-created_ node? As long as the node subscribing to the update topic is present, there shouldn't be an issue.

Reply:
It's a well-known issue in ROS that first messages on topics can get lost because subscribers did not setup their connections yet.